README.md

```diff
@@ -323,6 +323,10 @@ as `onNext`, `onComplete`, or `cancel`.
 
 `take%%<n>%%<id>` : The driver should hang the test thread until it has received at least `n` items IN TOTAL, and then immediately cancel.
 
+`disconnect` : The driver should disconnect the client from the server.
```

Comment:
What form is this? n.b. I'm not sure how much it matter but it seems like the error codes would differ.

Is this clean disconnect through API? If so, is it always client initiated?
Simulating Network disconnect?